src/components/panel/panel.js

```diff
@@ -1,4 +1,4 @@
 export default function errorPanel(panel) {
-  panel.setAttribute('tabIndex', 0);
+  panel.setAttribute('tabIndex', -1);
```

Comment:
We can just remove this line as the panel doesn't have an index value by default.

Reply:
Doesnt work if I remove that line because the panel is no longer focusable. So Ive removed it and used `scrollIntoView() `instead